### PR TITLE
Implemented the integrations component.

### PR DESCRIPTION
**Pull request by Skorpios604**

### CODE REVIEW

**Review by popnfresh234 — CHANGES_REQUESTED** [review]
There are some hardcoded dimension values that may give us some trouble down the road, I'm not sure how many of these are one off's or are values that we should be reusing throught the app.

If they are one offs no big deal, but if they are values that will be reused elsewhere we should think about refactoring them to the theme.

The one major issue I see is that we already have a `Button` component implemented, we should use that instead of coding up a new `Button` here as I believe this is just a standard button.

**Comment by Skorpios604** [comment]
Refactored code and addressed the comments. Please have a look. Thanks!

File: Client/src/Components/Integrations/index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            maxWidth: '300px',
+            wordWrap: 'break-word',
+            textAlign: 'left'
+          }}
+        >
+          This is a paragraph that provides more detail about the header.
+        </Typography>
+      </Box>
+      
+      <Button label="Click Me" level="primary" />
```
Comment (popnfresh234) [inline Client/src/Components/Integrations/index.jsx:51]: Sweet, I think this is the first component that uses one of our own components in it!  Nice work :+1:

**Review by popnfresh234 — APPROVED** [review]
Looks good to me!

File: Client/src/Components/Integrations/index.jsx (hunk 1 above)
Comment (Skorpios604) [reply Client/src/Components/Integrations/index.jsx:51]: Ya I noticed that too! Thanks!

**Pull request merged**